app/migrations/Version20180502142950.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Migrations;
+
+use AppBundle\Donation\DonationRequestUtils;
+use AppBundle\Entity\Donation;
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Ramsey\Uuid\Uuid;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class Version20180502142950 extends AbstractMigration implements ContainerAwareInterface
+{
+    use ContainerAwareTrait;
+
+    public function up(Schema $schema): void
+    {
+        if (!$this->container) {
```

Comment:
IMO this test is useless no ?

Reply:
Don't know It was to be sure I have a container set. But I can remove it